networkit/cpp/numerics/test/LAMGGTest.cpp

```diff
@@ -79,13 +77,9 @@ TEST_F(LAMGGTest, testSmallGraphs) {
         DEBUG("DONE");
 
     }
-
-    delete smoother;
 }
 
-
-
-Vector LAMGGTest::randVector(count dimension, double lower, double upper) const {
+Vector LAMGGTest::randVector(count dimension, double, double) const {
```

Comment:
This looks suspicious. Why doesn't the function use it's arguments?

Reply:
I don't know the details about that, I just removed unused constants.

Reply:
Looks like the originally wanted functionality was removed without removing the parameters.

Reply:
Possibly. In that case, remove the parameters as well.

Reply:
I removed them.